packages/core/src/common/keys.ts

```diff
@@ -252,7 +253,7 @@ export class KeyCode {
         }
         /* If duplicates i.e ctrl+ctrl+a or alt+alt+b or b+alt+b it is invalid */
         if (keys.length !== new Set(keys).size) {
-            throw new Error(`Can't parse keybinding ${keybinding} Duplicate modifiers`);
+            throw new Error(nls.localize('theia/core/keybinding/error.duplicateModifier', "Can't parse keybinding {0} Duplicate modifiers", keybinding));
```

Comment:
I don't believe we use `.` in our keys at the moment, so I'd update for consistency:

```suggestion
            throw new Error(nls.localize('theia/core/keybinding/duplicateModifierError', "Can't parse keybinding {0} Duplicate modifiers", keybinding));
```